cert/cert_test.go

```diff
@@ -1,111 +0,0 @@
-package cert
-
```

Comment:
If we're deleting `cert_test.go` (due to deleting `CompareCertificates()`) then we should also delete the files under the `cert/test_files` directory that are referenced in this file.